tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
+    def __init__(self):
```

Comment:
Once converted to a dataclass, this manually written `__init__` can be removed.  Use the generated method, which accepts arguments, to set all of the members when the class is constructed and inserted into the `chainData` class, per @oschwaldp-oci 's review comment.